Add GDPR

bakke92 opened the pull request:
https://github.com/bakke92/awesome-gdpr#readme

Privacy regulation in Europe. The General Data Protection Regulation (GDPR) is a law regulating data protection and privacy for all individuals in the EU and EEA. The regulation have increased focus and investments in privacy related research and work.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

PRs Reviewed:
[#1601 comment](https://github.com/sindresorhus/awesome/pull/1601#issuecomment-521332867)
[#1596 comment](https://github.com/sindresorhus/awesome/pull/1596#issuecomment-521335237)

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**


	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- **If you use a license badge, it should be SVG, not PNG.**
	- To verify that you've read all the guidelines, please add comment with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

bakke92 commented:
unicorn

sindresorhus commented:
> Your entry here should include a short description about the project/theme of the list. It should not describe the list itself. The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
✅ - [iOS](…) - Mobile operating system for Apple phones and tablets.
✅ - [Framer](…) - Prototyping interactive UI designs.
❌ - [iOS](…) - Resources and tools for iOS development.
❌ - [Framer](…)
❌ - [Framer](…) - prototyping interactive UI designs

bakke92 commented:
Thanks for the feedback, I have updated the entry to describe the theme of the list more thoroughly.

csabapalfi commented on readme.md:
```diff
@@ -464,6 +464,7 @@
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
 - [Cybersecurity Blue Team](https://github.com/meitar/awesome-cybersecurity-blueteam#readme) - Groups of individuals who identify security flaws in information technology systems.
 - [Fuzzing](https://github.com/cpuu/awesome-fuzzing#readme) - Automated software testing technique that involves feeding pseudo-randomly generated input data.
+- [GDPR](https://github.com/bakke92/awesome-gdpr#readme) - Privacy tools and resources.
```
Maybe something closer to what's in the repo is a bit better as a description:

> GDPR-compliant tools for website creators

Alternatively the acronyms could be expanded as well, maybe?

> European Union Global Data Protection Regulation (GDPR) compliant tools for website creators

bakke92 replied:
I agree that using the words "tools" would be describing the content. It is not merely just for website creators though. And the usage of "tools and resources" to describe isn't according to the requirements for Awesome repos when describing it in the readme document.
As I understand it is it OK in the Awesome-list though.

csabapalfi replied:
Ah, ok. Actually it makes sense. I now understand a bit better the requirements (upon re-reading the guidance).

sindresorhus replied:
> As I understand it is it OK in the Awesome-list though.

No, it should explain the theme, in this case GDPR, not the list itself.

Jogai commented:
The last link of your TOC is not working

bakke92 commented:
Thanks @Jogai. The wrong link is fixed.

nglgzz commented:
The text in first item in your TOC (Overview of the General Data Protection Regulation) doesn't match the heading it's linked to (Overview of GDPR). I suggest you expand the acronym on the heading to make them match.

bakke92 commented:
Thanks @nglgzz, fixed it as suggested.

sindresorhus commented:
I think you can elaborate more in your readme about GDPR than just `Privacy regulation in Europe.`.

---

`Overview of the General Data Protection Regulation` This is not the place to elaborate what the acronym means. This should simply be `Overview`.

---

> The image should link to the project website or any relevant website.

---

```
❯ awesome-lint 'https://github.com/bakke92/awesome-gdpr'
✖ Linting

  README.md:9:3
  ✖   9:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  10:1   ToC item "Guidelines" link "#Guidelines" not found    remark-lint:awesome/toc
  ✖  10:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  11:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  12:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  13:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  14:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  15:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  16:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  19:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  20:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  23:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  24:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  27:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  28:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  29:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  30:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  33:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  34:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  35:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  38:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  39:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  40:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  41:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  42:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  42:67  List item description must start with valid casing    remark-lint:awesome/list-item
  ✖  45:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  46:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  47:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  48:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  49:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  50:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  51:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  52:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  56:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  57:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  58:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  59:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  60:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  61:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  62:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  63:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  66:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  67:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  68:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  69:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  70:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  71:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  74:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent
  ✖  75:3   Incorrect indentation before bullet: remove 2 spaces  remark-lint:list-item-bullet-indent

  50 errors
```

https://github.com/sindresorhus/awesome-lint

bakke92 commented:
Thanks, @sindresorhus! 
Updated the description to better describe GDPR.
Changed "Overview" to "Legal text" as this better describes what the content is.
Linked the image to EU website on GDPR.
Fixed linter errors (hopefully all of them as I can't run the linter myself on my computer).

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1177083395991453697
